### DISCUSSION TITLE
Bypass Auto Login

### DISCUSSION THREAD

**Discussion by Stonebkfly** [oidc-wp/openid-connect-generic#327] [Q&A] [answered]
I have a WP site that uses the openid-connect-generic plugin. It is setup with auto login. When I login with the SSO user I get the following message : "Application not configured." Is there a way I can bypass the auto login so I can get to the wp-login page and login using username and password?
Thank you!
Joseph

**Comment by timnolte** [comment, marked as answer]
> I have a WP site that uses the openid-connect-generic plugin. It is setup with auto login. When I login with the SSO user I get the following message : "Application not configured." Is there a way I can bypass the auto login so I can get to the wp-login page and login using username and password?
> Thank you!
> Joseph

@Stonebkfly if you can access your site files on the server you could change the permissions on the plugin folder so that it is no longer accessible and that will cause the plugin to be deactivated. If your hosting has the WP-CLI on the server you can use that to deactivate the plugin properly.

**Reply by Stonebkfly** [reply]
Thank you Tim! That is good to know!
